### ISSUE TITLE
Asking about Testing period

### ISSUE THREAD

**Issue by NathanielNguyen11** [jakeret/tf_unet#124]
Dear Mr/Ms
Firstly, thank for your software very much, with this one i have learned a lot of thing about deep learning.
Secondly, I write this topic to kindly ask about the output of testing stage.
Because, when i take the result, it's was cool with strange color.
This is from the training process.
![epoch_99](https://user-images.githubusercontent.com/32150689/32205180-3058fe68-be31-11e7-828f-329264bd9f7f.jpg)
This is from the testing process
![problem0](https://user-images.githubusercontent.com/32150689/32205181-308072ea-be31-11e7-9d0e-2ec3efc00e2f.png)

Thank for you attention very much

**Comment by jakeret** [comment]
Isn't this just a plotting issue? During the training `tf_unet` uses the Pillow (`PIL`) package whereas the last plot looks like matplotlib

**Comment by abbyDC** [comment]
hello @NathanielNguyen11 ! I'm trying to do the same image segmentation. may I know what code you used? is it similar to the demo toy problem or you made your own?

**Comment by NathanielNguyen11** [comment]
@jakeret . Dear mr Jakeret
First of all, Thank for your answer very much.
You mean, "I should use plot (using  command from PIL package) at the last testing period, right? "

**Comment by NathanielNguyen11** [comment]
@abbyDC  hi,
Actually , I just used it by the usage of Mr Jakeret. but i think that you must read the "ImageDataProvider function " before using it.

**Comment by jakeret** [comment]
You can use either of both packages to create the output images. 
In your particular case, I think that the weird colors in matplotlib are just a matter of value scale. There is certainly good documentation on this matter on the matplotlib site

**Comment by NathanielNguyen11** [comment]
@jakeret  thank a lot , it makes sense with me alot.

**Comment by abbyDC** [comment]
Thanks for your answer @NathanielNguyen11! Did you just change the part where the images were read? I read my images like this
`generator = image_util.ImageDataProvider('dataset/*', data_suffix=".png", mask_suffix='_mask.png')`
and haven't changed the other parts of the code from the demo_toy_problem from jakeret. May I know if yours is similar to mine? If not, may I see how you did it? Thank you!

**Comment by NathanielNguyen11** [comment]
@abbyDC actually , I thought that i i change something from the original code if you want, here is may email: ngocquang870@gmail.com
if you want i can send it to you

**Comment by abbyDC** [comment]
Hello @NathanielNguyen11! I think I'm now getting the output I expected. I just changed the batch size and did batch normalization during training. I truly appreciate your effort in replying. Thank you very much! :D
